<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuPage</class>
 <widget class="QMainWindow" name="MenuPage">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>952</width>
    <height>643</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Portable Electronic Target (P.E.T) - Main Page</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::TabShape::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_hit_area">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="tabletTracking">
     <bool>true</bool>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::Box</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="bulletCount_lbl">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>90</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel 
{

color: black;
}</string>
    </property>
    <property name="text">
     <string>Bullet Count's</string>
    </property>
   </widget>
   <widget class="QLabel" name="targetScorelbl">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>221</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>true</italic>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel 
{

color: black;
}</string>
    </property>
    <property name="text">
     <string>Hit scores: </string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="targetScorelbl_2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>288</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel 
{

color: black;
}</string>
    </property>
    <property name="text">
     <string>Bullet Hit Logs:</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>70</y>
      <width>241</width>
      <height>501</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QGraphicsView
{
Background: rgb(104, 125, 147);

}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="interactive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="totalScoreDisplay">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>248</y>
      <width>201</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
 background-color: white;
color:black;

}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_bulletCount">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>114</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::EchoMode::Normal</enum>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_reset">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Start">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_no1T">
    <property name="geometry">
     <rect>
      <x>365</x>
      <y>128</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>501</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Welcome to Portable Electronic Training</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_no2T">
    <property name="geometry">
     <rect>
      <x>365</x>
      <y>148</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_no3T">
    <property name="geometry">
     <rect>
      <x>365</x>
      <y>168</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_no4T">
    <property name="geometry">
     <rect>
      <x>365</x>
      <y>188</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_no5T">
    <property name="geometry">
     <rect>
      <x>365</x>
      <y>208</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_no6T">
    <property name="geometry">
     <rect>
      <x>365</x>
      <y>228</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>310</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_no1Left">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>310</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_no3B">
    <property name="geometry">
     <rect>
      <x>365</x>
      <y>470</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_no2B">
    <property name="geometry">
     <rect>
      <x>365</x>
      <y>490</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_no6B">
    <property name="geometry">
     <rect>
      <x>365</x>
      <y>410</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_no4B">
    <property name="geometry">
     <rect>
      <x>365</x>
      <y>450</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_no1B">
    <property name="geometry">
     <rect>
      <x>365</x>
      <y>510</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_no5B">
    <property name="geometry">
     <rect>
      <x>365</x>
      <y>430</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_no1Left_2">
    <property name="geometry">
     <rect>
      <x>195</x>
      <y>310</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_no1Left_3">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>310</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_no1Left_4">
    <property name="geometry">
     <rect>
      <x>235</x>
      <y>310</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_no1Left_5">
    <property name="geometry">
     <rect>
      <x>255</x>
      <y>310</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_no1Left_6">
    <property name="geometry">
     <rect>
      <x>275</x>
      <y>310</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_no2R">
    <property name="geometry">
     <rect>
      <x>535</x>
      <y>310</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_no3R">
    <property name="geometry">
     <rect>
      <x>515</x>
      <y>310</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_no4R">
    <property name="geometry">
     <rect>
      <x>495</x>
      <y>310</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_no6R">
    <property name="geometry">
     <rect>
      <x>455</x>
      <y>310</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_no5R">
    <property name="geometry">
     <rect>
      <x>475</x>
      <y>310</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_no1R">
    <property name="geometry">
     <rect>
      <x>555</x>
      <y>310</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Mouse_Area">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>151</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel 
{

color: black;
}</string>
    </property>
    <property name="text">
     <string>Mouse Area </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_mouse_pos">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>184</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::EchoMode::Normal</enum>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="coodLabel">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>694</x>
      <y>190</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>X = 0, Y = 0</string>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>540</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Status: 0 Initial Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="bCountDisplay">
    <property name="geometry">
     <rect>
      <x>698</x>
      <y>120</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0 / 20</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lstLogs">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>320</y>
      <width>201</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>TYPE B</string>
    </property>
   </widget>
   <zorder>graphicsView</zorder>
   <zorder>lbl_hit_area</zorder>
   <zorder>bulletCount_lbl</zorder>
   <zorder>targetScorelbl</zorder>
   <zorder>targetScorelbl_2</zorder>
   <zorder>totalScoreDisplay</zorder>
   <zorder>line_bulletCount</zorder>
   <zorder>btn_reset</zorder>
   <zorder>btn_Start</zorder>
   <zorder>label_no1T</zorder>
   <zorder>label_2</zorder>
   <zorder>label_no2T</zorder>
   <zorder>label_no3T</zorder>
   <zorder>label_no4T</zorder>
   <zorder>label_no5T</zorder>
   <zorder>label_no6T</zorder>
   <zorder>label</zorder>
   <zorder>label_no1Left</zorder>
   <zorder>label_no3B</zorder>
   <zorder>label_no2B</zorder>
   <zorder>label_no6B</zorder>
   <zorder>label_no4B</zorder>
   <zorder>label_no1B</zorder>
   <zorder>label_no5B</zorder>
   <zorder>label_no1Left_2</zorder>
   <zorder>label_no1Left_3</zorder>
   <zorder>label_no1Left_4</zorder>
   <zorder>label_no1Left_5</zorder>
   <zorder>label_no1Left_6</zorder>
   <zorder>label_no2R</zorder>
   <zorder>label_no3R</zorder>
   <zorder>label_no4R</zorder>
   <zorder>label_no6R</zorder>
   <zorder>label_no5R</zorder>
   <zorder>label_no1R</zorder>
   <zorder>lbl_Mouse_Area</zorder>
   <zorder>line_mouse_pos</zorder>
   <zorder>coodLabel</zorder>
   <zorder>statusLabel</zorder>
   <zorder>bCountDisplay</zorder>
   <zorder>lstLogs</zorder>
   <zorder>pushButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>952</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_reset</sender>
   <signal>clicked()</signal>
   <receiver>lstLogs</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>45</x>
     <y>585</y>
    </hint>
    <hint type="destinationlabel">
     <x>748</x>
     <y>524</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
